node/tty.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+// This module implements 'tty' module of Node.JS API.
+// ref: https://nodejs.org/api/tty.html
+
+// Returns true when the given numeric fd is associated with a TTY and false otherwise.
+function isatty(fd: unknown) {
```

Comment:
```suggestion
function isatty(fd: number) {
```
I think we can make this a number since the [node docs](https://nodejs.org/api/tty.html#tty_tty_isatty_fd) also state it as a number. Thoughts?

Reply:
I checked DefinitelyTyped as well and they also type this as `number`.  ref: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/cd10dd7/types/node/tty.d.ts#L8

It seems better to make it `number`. Thank you for you suggestion!